.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </state>
+</component>
```

Comment:
All this .idea stuff shouldn't be added to the patch. Hint: you can add this to your global `.gitgnore` file.

Reply:
sure, my bad. Forgot to check.